model/postgres/postgres_resource.rb

```diff
@@ -13,6 +13,7 @@ class PostgresResource < Sequel::Model
   one_to_many :firewall_rules, class: PostgresFirewallRule, key: :postgres_resource_id
   one_to_many :metric_destinations, class: PostgresMetricDestination, key: :postgres_resource_id
   many_to_one :private_subnet
+  many_to_one :firewall
```

Comment:
Do we really need to keep the firewall as seperate column? Every piece of data we add increases the cost of data tracking. Can't we directly use `private_subnet.firewalls.first` instead?